src/main/channels/audio/DeleteOneAudioChannel.ts

```
@@ -1,20 +1,18 @@
 import { IpcChannel } from '@main/interfaces/IpcChannel';
+import { getUserAudioPath } from '@main/utils/path';
 import { DELETE_ONE_AUDIO_CHANNEL } from '@shared/channels';
 import { DeleteOneAudioParams } from '@shared/interfaces/Audio';
 import { IpcRequest } from '@shared/interfaces/IpcRequest';
 import { IpcMainEvent } from 'electron';
 import log from 'electron-log/main';
-import * as fs from 'fs';
-import * as path from 'path';
+import fs from 'fs';
+import path from 'path';
```

ESLint

[error] 'path' is defined but never used.

(@typescript-eslint/no-unused-vars)